packages/core/src/websockets/reconnecting-websocket.ts

```diff
@@ -164,8 +165,7 @@ export type Options<WS extends IWebSocket = WebSocket> = {
   maxRetries?: number;
   maxEnqueuedMessages?: number;
   startClosed?: boolean;
-  debug?: boolean;
-  debugLogger?: (...args: any[]) => void;
+  logger?: ILogger;
```

Comment:
Can we avoid this breaking change?

Core is not the only consumer of this class

Reply:
We might want to support both `logger` and set `debugLogger` to `logger` if `logger` exists but `debugLogger` wasn't passed in